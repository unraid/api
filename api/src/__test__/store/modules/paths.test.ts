import { expect, test } from 'vitest';

import { store } from '@app/store/index.js';

test('Returns paths', async () => {
    const { paths } = store.getState();
    expect(Object.keys(paths)).toMatchInlineSnapshot(`
		[
		  "core",
		  "unraid-api-base",
		  "unraid-data",
		  "docker-autostart",
		  "docker-socket",
		  "parity-checks",
		  "htpasswd",
		  "emhttpd-socket",
		  "states",
		  "dynamix-base",
		  "dynamix-config",
		  "myservers-base",
		  "myservers-config",
		  "myservers-config-states",
		  "myservers-env",
		  "myservers-keepalive",
		  "keyfile-base",
		  "machine-id",
		  "log-base",
		  "unraid-log-base",
		  "var-run",
		  "auth-sessions",
		  "auth-keys",
		]
	`);
});
