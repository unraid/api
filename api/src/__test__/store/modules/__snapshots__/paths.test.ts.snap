// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Returns paths 1`] = `
[
  "core",
  "unraid-api-base",
  "unraid-data",
  "docker-autostart",
  "docker-socket",
  "parity-checks",
  "htpasswd",
  "emhttpd-socket",
  "states",
  "dynamix-base",
  "dynamix-config",
  "myservers-base",
  "myservers-config",
  "myservers-config-states",
  "myservers-env",
  "myservers-keepalive",
  "keyfile-base",
  "machine-id",
  "log-base",
  "unraid-log-base",
  "var-run",
  "auth-sessions",
  "auth-keys",
  "passwd",
  "libvirt-pid",
  "activationBase",
  "webGuiBase",
  "identConfig",
  "activation",
  "boot",
  "webgui",
]
`;
